sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_common/_schema.py

```diff
@@ -50,13 +50,12 @@ class SchemaProperties(object):
 
     def __init__(
         self,
-        id=None,    # pylint:disable=redefined-builtin
         **kwargs
     ):
-        # type: (Optional[str], Any) -> None
-        self.id = id
+        # type: (Any) -> None
+        self.id = kwargs.pop('id')
         self.format = kwargs.get('format')
```

Comment:
nit: I don't either pop or get is fine, but it's better to stay consistent within our code
```suggestion
        self.format = kwargs.pop('format')
```

Reply:
good catch!